test/celeritas/CMakeLists.txt

```diff
@@ -235,7 +235,7 @@ else()
   set(_fieldprop_filter)
 endif()
 
-celeritas_add_test(field/Fields.test.cc
+celeritas_add_device_test(field/Fields
```

Comment:
Should this need a "DISABLED" if not `CELERITAS_USE_covfie`?

Reply:
Can we disable only the device test?